wikimedia/wikipedia-ios
# Filter out plug-in WKWebView log from displayed errors

**Phabricator:** T374566

This "plug-in handled load" error is less an actual error than a log which indicates that WebKit relayed the handling of media content (e.g. mp3) to a plug-in. This PR is a suggestion to ensure that the log won't be displayed to the user.

### Notes
I wanted to ensure that it was not an error with the VC's WKWebView setup so I created an empty app with just the following code. It was enough to trigger the message, which suggests that this is expected behavior when loading a mp3 in a WKWebView.

```
class ViewController: UIViewController, WKNavigationDelegate {

  override func viewDidLoad() {
    super.viewDidLoad()
    let webView = WKWebView(frame: view.frame)
    webView.navigationDelegate = self
    view.addSubview(webView)
    webView.load(URLRequest(url: URL(string: "https://www.soundhelix.com/examples/mp3/SoundHelix-Song-3.mp3")!))
  }

  func webView(_ webView: WKWebView, didFail navigation: WKNavigation!, withError error: Error) {
  // Will print "plug-in handled load"
    print("\(error.localizedDescription)")
  }
}
```

### Test Steps
1. Launch the app
2. Go to any article where the title has audio playback (e.g. Pablo Neruda)
3. "Plug-in handled load" banner should not be displayed anymore
